prow/gerrit/reporter/reporter.go

```diff
@@ -118,9 +119,13 @@ func (c *Client) Report(pj *v1.ProwJob) error {
 	clientGerritRevision := client.GerritRevision
 	clientGerritID := client.GerritID
 	clientGerritInstance := client.GerritInstance
+	pjTypeLabel := kube.ProwJobTypeLabel
 
-	// list all prowjobs in the patchset
-	selector := labels.Set{clientGerritRevision: pj.ObjectMeta.Labels[clientGerritRevision]}
+	// list all prowjobs in the patchset matching pj's type (pre- or post-submit)
+	selector := labels.Set{
+		clientGerritRevision: pj.ObjectMeta.Labels[clientGerritRevision],
+		pjTypeLabel:          pj.ObjectMeta.Labels[pjTypeLabel],
```

Comment:
maybe also add to the selector in `ShouldReport` as well? (although that should not happen, but just in case)

Reply:
👍